src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -441,6 +484,52 @@ public void largeDataValidationTest(final String inputVcfName,
         System.out.println("Total Elapsed Time: " + (endTime - overallStartTime) / 1e9 + "s");
     }
 
+    @Test(dataProvider = "provideForNonTrivialLargeDataTest")
+    public void nonTrivialLargeDataTest(final String inputVcfName,
+                               final String referencePath,
+                               final String referenceVersion,
+                               final String dataSourcesPath,
+                               final String expectedOutputPath ) {
+
+        final File outputFile = createTempFile(tmpOutDir + File.separator + inputVcfName + ".funcotator", ".vcf");
+
+        final ArgumentsBuilder arguments = createBaselineArgumentsForFuncotator(
+                inputVcfName,
+                outputFile,
+                referencePath,
+                dataSourcesPath,
+                referenceVersion,
+                FuncotatorArgumentDefinitions.OutputFormatType.VCF,
```

Comment:
Might want to mirror each VCF test case here with a MAF output test, since this is the "main" integration test for Funcotator, and will likely be expanded over time.

Reply:
Sure.  This seems somewhat redundant given that there is a test for VCF/MAF concordance already, but I'll add in a MAF check.